packages/main-api/key.sh

```
@@ -0,0 +1,22 @@
+    make .build/evm-keys > /dev/null 2>&1
+    cd $cdir
+fi
+
+# if jq is not installed, install it
+if ! command -v jq &> /dev/null
+then
+    echo "jq could not be found, install it..."
+    exit 1
+fi
+
+export ADDR=$(jq ".[0].address" ../../.build/evm-keys.json | tr -d \")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)